src/main/resources/RecruitmentApp/Configuration.xml

```diff
@@ -1,182 +1,283 @@
-   '_actions' : { "actions" : ["delete"] }
-})
-#set($officeTableColumns = ["office", "_actions"])
-#livetable("officeTable" $officeTableColumns $officeTableColumnsProperties $officeTableOptions)
-{{html clean="false"}}
-&lt;form method="post" action=""&gt;
-   Office: &lt;input type="text" name="officeName" /&gt;
-   &lt;input type="submit" value="Add" class="button" /&gt;
-&lt;/form&gt;
-{{/html}}
-
-###Display status
-== Candidate Status ==
-#set($statusTableOptions = {
-  'className' : 'RecruitmentApp.StatusClass',
-  'translationPrefix' : 'recruitmentapp.status.livetable.',
-  'rowCount': 10
-})
-#set($statusTableColumnsProperties = {
-  'status' : { "type" : "String" },
-  '_actions' : { 'actions': ['delete']}
-})
-#set($statusTableColumns = ['status', '_actions'])
-#livetable('statusTable' $statusTableColumns $statusTableColumnsProperties $statusTableOptions)
-{{html clean="false"}}
-&lt;form method="post" action=""&gt;
-   Status: &lt;input type="text" name="statusName" /&gt;
-   &lt;input type="submit" value="Add" class="button" /&gt;
-&lt;/form&gt;
-{{/html}}
-
-###Display source
-== Source ==
-#set($sourceTableOptions = {
-   'className': 'RecruitmentApp.SourceClass',
-   'translationPrefix': 'recruitmentapp.source.livetable.',
-   'rowCount' : 10
-})
-#set($sourceTableColumnsProperties = {
-   'source': { "type": "String" },
-   '_actions': { 'actions' : ['delete'] }
-})
-#set($sourceTableColumns = ['source', '_actions'])
-#livetable('sourceTable' $sourceTableColumns $sourceTableColumnsProperties $sourceTableOptions)
-{{html clean="false"}}
-&lt;form method="post" action=""&gt;
-   Source: &lt;input type="text" name="sourceName" /&gt;
-   &lt;input type="submit" value="Add" class="button" /&gt;
-&lt;/form&gt;
-{{/html}}
-
-###Display next steps
-== Next Steps ==
-#set($stepTableOptions = {
-    'className': 'RecruitmentApp.NextStepClass',
-    'translationPrefix' : 'recruitmentapp.nextstep.livetable.',
-    'rowCount' : 10
-})
-#set($stepTableColumnsProperties = {
-    'step' : { "type" : "String" },
-    '_actions': { 'actions': ['delete'] }
-})
-#set($stepTableColumns = ['step', '_actions'])
-#livetable('stepTable' $stepTableColumns $stepTableColumnsProperties $stepTableOptions)
-{{html clean="false"}}
-&lt;form method="post" action=""&gt;
-   Step: &lt;input type="text" name="stepName"/&gt;
-   &lt;input type="submit" value="Add" class="button" /&gt;
-&lt;/form&gt;
-{{/html}}
-
-###import
-== Import ==
-Specify the space where the old recruitment application is installed.
-{{html clean="false"}}
-&lt;form method="post" action="$xwiki.getURL("RecruitmentApp.MigrationScript")"&gt;
-   &lt;input type="hidden" name="doc" value="$doc" /&gt;
-   &lt;input type="hidden" name="query" value="$request.getQueryString()" /&gt;
-   Space: &lt;input type="text" name="importSpace" value="$importConfigObj.space"&gt;
-   &lt;input type="submit" value="Import" class="button"/&gt;
-&lt;/form&gt;
-{{/html}}
+  <hidden>false</hidden>
```

Comment:
Technical pages need to be "Hidden". @9inpachi would be great if you could also take a look at the Best Practices documentation for Applications: https://dev.xwiki.org/xwiki/bin/view/Community/ApplicationDevelopmentBestPractices

Reply:
I made sure all the pages were hidden.
I guess this got overlooked. Thanks for pointing out.

Reply:
This is rather odd. I had this as "true" in my fork and still have.